src/odb/src/swig/common/swig_common.cpp

```diff
@@ -298,6 +299,18 @@ void createSBoxes(odb::dbSWire* swire,
     odb::dbSBox::create(swire, via, point.getX(), point.getY(), type);
 }
 
+void createPGpins(odb::dbBlock* block, odb::dbTech* tech, const char* source_net_name, int num_connection_points, std::string position)
+{
+  odb::dbPGpin PGpin;
+  PGpin.create_PGpin(block, tech, source_net_name, num_connection_points, position);
+}
+
+void p2proute(odb::dbBlock* block, const char* nett, const char* instt, const char* itermm)
```

Comment:
This doesn't do any routing.  It is more about building a connection.

Reply:
name changed